GVFS/GVFS.FunctionalTests/Settings.cs

```diff
@@ -47,6 +47,8 @@ public static void Initialize()
                     string root = "/GVFS.FT";
                     EnlistmentRoot = Path.Combine(root, "test");
                     ControlGitRepoRoot = Path.Combine(root, "control");
+                    FastFetchRoot = Path.Combine(root, "FFtest");

```

Comment:
Please don't abbreviate the folder names

Reply:
And I would try to keep a consistent naming pattern as we do for the Windows tests (see line 40)